db/migrate/20250306110910_remove_server_states.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class RemoveServerStates < ActiveRecord::Migration[8.0]
+  def change
+    revert do
+      create_table :server_states do |t|
+        t.string :name
+
+        t.timestamps null: false
```

Comment:
```suggestion
        t.timestamps
```

I think it's the defaults